.travis.yml

```diff
@@ -4,7 +4,7 @@
 # Copyright (c) 2013 Mateusz Loskot <mateusz@loskot.net>
 #
 language: cpp
-
+sudo: required
```

Comment:
@vadz I think that the missing `sudo` requirement was an overlook. All of Travis CI shell scripts require `sudo` and I'm actually surprised `install_*.sh` scripts worked without `sudo: required`.
Am I missing something?